source/developer/interactive-messages.rst

```diff
@@ -301,7 +301,7 @@ Context
         }
       }
 
-  Then, when the integration receives the request, it can act based on the action ID.
+  Then, when the integration receives the request, it can act based on the action ID, which should match the `[a-zA-Z0-9]+` RegExp pattern.
```

Comment:
Just want to check whether this should be "RegExp" or "RegEx" and whether it's a separate pattern to "regex". Thank you!

Reply:
I was thinking of regular expression so regex would just fine. I might have been under [influence](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp) here :sweat_smile: 
As you prefer :smiley:

Reply:
The term "regex" is the one I'm personally more familiar with. However, Google uses the term "RegExp". To avoid unintentional ambiguity, perhaps going with "regular expression" would work? 

@justinegeffen - Thoughts?

Reply:
I've replaced `RegExp pattern` with `regular expression` in https://github.com/mattermost/docs/commit/69b1b923761bd441f5c84aeafe42f2161cd2a5c3. 
Please let me know if you would like to see something else instead.

Reply:
That works for me, thank you @thierrymarianne! And thanks for clarifying, @cwarnermm.